compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_black_box.rs

```diff
@@ -1,18 +1,19 @@
 use acvm::{
     acir::{brillig::BlackBoxOp, BlackBoxFunc},
-    FieldElement,
+    AcirField,
 };
 
 use crate::brillig::brillig_ir::{
     brillig_variable::{BrilligVariable, BrilligVector, SingleAddrVariable},
+    debug_show::DebugToString,
     BrilligBinaryOp, BrilligContext,
 };
 
 /// Transforms SSA's black box function calls into the corresponding brillig instructions
 /// Extracting arguments and results from the SSA function call
 /// And making any necessary type conversions to adapt noir's blackbox calls to brillig's
-pub(crate) fn convert_black_box_call(
-    brillig_context: &mut BrilligContext,
+pub(crate) fn convert_black_box_call<F: AcirField + DebugToString>(
```

Comment:
`AcirField` already includes `Debug` and `Display`: can `DebugToString` also be included? It might require moving `DebugToString` or another larger change, in which case better to save for another PR.

Reply:
This would end up being a larger change so I think we can hold off for now.